rover/config/config.js

```diff
@@ -7,9 +7,9 @@ module.exports = {
   database: process.env.DB_NAME,
   synchronize: true,
   logging: false,
-  entities: ['src/entity/**/*.ts'],
-  migrations: ['src/migration/**/*.ts'],
-  subscribers: ['src/subscriber/**/*.ts'],
+  entities: ['build/entity/**/*.js'],
+  migrations: ['build/migration/**/*.ts'],
```

Comment:
is this supposed to remain a .ts?

Reply:
it will probably need to change in the future, but currently there are no migrations